src/main/java/htsjdk/samtools/SAMFileHeader.java

```diff
@@ -48,13 +49,14 @@
     public static final String GROUP_ORDER_TAG = "GO";
     public static final String CURRENT_VERSION = "1.5";
     public static final Set<String> ACCEPTABLE_VERSIONS =
-            new HashSet<String>(Arrays.asList("1.0", "1.3", "1.4", "1.5"));
+            new HashSet<>(Arrays.asList("1.0", "1.3", "1.4", "1.5"));
```

Comment:
Well, if you're going to start cleaning up, this could be `CollectionUtil.makeSet(....)`